src/main/java/org/broadinstitute/hellbender/tools/spark/sv/StructuralVariationDiscoveryArgumentCollection.java

```diff
@@ -55,11 +59,11 @@
 
         @Argument(doc = "Minimum weight of the corroborating read evidence to validate some single piece of evidence.",
                 fullName = "min-evidence-count")
```

Comment:
We should change the name of the args away from `count` now that they are scaled. I'm blanking a little on what the best names would be.. `min-evidence-coverage-fraction` or something? Also, please update the `doc` string to be a little more descriptive of the new value (and maybe include a small calculation showing that the default value is equivalent to 15 reads at 42X coverage or something?)

Reply:
Done.